src/src/com/microsoft/aad/adal/AuthenticationConstants.java

```diff
@@ -340,6 +340,8 @@
     }
 
     public static final String ADAL_PACKAGE_NAME = "com.microsoft.aad.adal";
+    
+    public static final String FIRST_PARTY_FAMILY_ID = "1";
```

Comment:
Can we call this MS_FAMILY_ID instead of referring to it as 1st party.  It's clearer to a wider audience what MS is vs 1st party.